packages/studio-base/src/components/Chart/types.ts

```diff
@@ -18,6 +18,9 @@ type Datum = ScatterDataPoint & {
   value?: string | number | bigint | boolean;
   // Constant name for the datum (used by state transitions)
   constantName?: string | undefined;
+
+  // Used only for showing multiple states in state transition
```

Comment:
```suggestion
  // The states in a state transition
```

I tend to prefer docs that say what the thing is or what it should contain because the use might evolve over time.